boefjes/boefjes/plugins/kat_sec_txt/main.py

```diff
@@ -0,0 +1,19 @@
+import json
+from typing import List, Tuple, Union
+
+from sectxt import SecurityTXT
+
+from boefjes.job_models import BoefjeMeta
+
+
+def run(boefje_meta: BoefjeMeta) -> List[Tuple[set, Union[bytes, str]]]:
+    input_ = boefje_meta.arguments["input"]
+    host = input_["name"]
+
+    secTXT = SecurityTXT(str(host))
+    results = {
+        "valid": secTXT.is_valid(),
+        "errors": secTXT.errors,
+    }
+
+    return [(set(), json.dumps(results))]
```

Comment:
We should add a MIME type here.

Maybe it would make more sense to retrieve the `security.txt` file as a raw file instead of directly validating it in the Boefje itself?

Reply:
It would, however since there are multiple url's that might host the security.txt and the location does matter for the result (there might even be two), this complicates this by a bit.

Reply:
According to RFC9116 `/.well-known/security.txt` must be used. The RFC also has some text about legacy `/security.txt`, but I think we should check whether the RFC is implemented and for that we only need to fetch /.well-known/security.txt` .

But given that the boefje/normalizer currently works as is, I would prefer to merge the PR and improve this later.

Reply:
Agreed, however, if we do find a legacy security.txt we can still check it, *and* raise an issue with its location.